### PR TITLE
finished changing apiauth to sheetsClient and changed favicon

### LINKED ISSUE #7: Change the input for db functions from apiAuthClient to the prebuilt sheets API object
in /api/server.js

- [x] Add sheetsClient to main function
- [x] Refactor all functions with apiAuthClient to take in sheetsClient
- [x] Change all endpoints calling functions requiring deprecated apiAuthClient to pass in sheetsClient instead

### PR DESCRIPTION
**Pull request by adityaiyer7**
closes #7 , #6

### CODE REVIEW

File: api/server.js
Comment (Connor-Bernard) [inline api/server.js:1]: Lets remove the commented out sheetsclient in every function

File: website/src/components/BinTable.js
Comment (Connor-Bernard) [inline website/src/components/BinTable.js:1]: Check this back out to main

**Review by Connor-Bernard — CHANGES_REQUESTED** [review]
Couple of changes on files.  Nice job otherwise :)

**Review by Connor-Bernard — CHANGES_REQUESTED** [review]
I made a couple of changes for formatting and documentation.  We should move the favicon to it's own branch and PR instead of having a joined PR for the two features, but after that, this all looks good.

**Comment by adityaiyer7** [comment]
I've moved the favicon to it's own branch and created a new PR for that branch. I don't see any additional comments other than that so I'm requesting a re-review on this PR.

**Pull request closed without merging**